inception-ui-kb/src/main/java/de/tudarmstadt/ukp/inception/ui/kb/feature/ConceptFeatureEditor.java

```diff
@@ -117,59 +115,86 @@ public void onConfigure(JQueryBehavior behavior)
 
             @Override
             protected IJQueryTemplate newTemplate() {
-                return KendoChoiceDescriptionScriptReference.template();
+                return new IJQueryTemplate() {
+                    private static final long serialVersionUID = 8656996525796349138L;
+
+                    @Override
+                    public String getText()
+                    {
+                        StringBuilder sb = new StringBuilder();
+                        sb.append("<div style=\"max-width: 450px\">");
+                        sb.append("  <div class=\"item-title\">");
+                        sb.append("    ${ data.name }");
+                        sb.append("  </div>");
+                        sb.append("  <div class=\"item-identifier\">");
+                        sb.append("    ${ data.identifier }");
+                        sb.append("  </div>");
+                        sb.append("  <div class=\"item-description\">");
+                        sb.append("    ${ data.description }");
+                        sb.append("  </div>");
+                        if (DEVELOPMENT.equals(getApplication().getConfigurationType())) {
+                            sb.append("  <div class=\"item-description\">");
+                            sb.append("    ${ data.debugInfo }");
+                            sb.append("  </div>");
+                        }
+                        sb.append("</div>");
+                        return sb.toString();
+                    }
+
+                    @Override
+                    public List<String> getTextProperties()
+                    {
+                        List<String> properties = new ArrayList<>();
+                        properties.add("name");
+                        properties.add("identifier");
+                        properties.add("description");
+                        if (DEVELOPMENT.equals(getApplication().getConfigurationType())) {
+                            properties.add("debugInfo");
+                        }
+                        return properties;
+                    }
+                    
+                };
             }
         };
 
         return field;
     }
+    
+    private List<KBHandle> getEntities(AnnotatorState aState, AnnotationActionHandler aHandler,
```

Comment:
Is this method getting candidate concepts?

Reply:
Renamed.